package.json

```diff
@@ -39,13 +39,15 @@
   ],
   "dependencies": {
     "axios": "^1.2.2",
-    "jsonschema": "^1.4.1"
+    "json2csv": "^6.0.0-alpha.2",
+    "jsonschema": "^1.4.1",
+    "typescript": "^5.1.6"
```

Comment:
Can you add `typescript` as a dev dependency using `--save-dev`? It is not required to run your code.

Reply:
@vinaysingh8866